django/db/models/fields/__init__.py

```diff
@@ -1567,10 +1567,8 @@ def format_number(self, value):
         """
         # Method moved to django.db.backends.utils.
         #
-        # It is preserved because it is used by the oracle backend
-        # (django.db.backends.oracle.query), and also for
-        # backwards-compatibility with any external code which may have used
-        # this method.
+        # It is preserved for backwards-compatibility with any external code
```

Comment:
This has been around since Django 1.0 (b3b71a0922334c70bbc646a4873010f808196671). I think the method should be fine to remove if Django no longer uses it. If we get complaints about it, we can add it back or start a deprecation.